sw/host/opentitanlib/src/dif/rstmgr.rs

```diff
@@ -26,6 +27,26 @@ pub enum RstmgrReg {
     ErrCode = bindgen::dif::RSTMGR_ERR_CODE_REG_OFFSET,
 }
 
+/// BitFields for the CPU_INFO_CTRL register.
+pub struct RstmgrCpuRegwen;
+
+impl RstmgrCpuRegwen {
+    pub const EN: u32 = 0b1 << bindgen::dif::RSTMGR_CPU_REGWEN_EN_BIT;
+}
+
+/// BitFields for the CPU_INFO_CTRL register.
+pub struct RstmgrCpuInfoCtrl;
+
+impl RstmgrCpuInfoCtrl {
+    pub const EN: u32 = 0b1 << bindgen::dif::RSTMGR_CPU_INFO_CTRL_EN_BIT;
+
+    pub const INDEX: BitField = BitField {
+        offset: bindgen::dif::RSTMGR_CPU_INFO_CTRL_INDEX_OFFSET,
+        // Relies on mask being continuous
+        size: bindgen::dif::RSTMGR_CPU_INFO_CTRL_INDEX_MASK.count_ones(),
+    };
+}
+
```

Comment:
It's up to you if you think it's worth it, but you could add a compile-time assert for the mask being contiguous. I *think* that something like this would work?

```suggestion

const _: () = {
    assert!(RSTMGR_CPU_INFO_CTRL_INDEX_MASK.wrapping_add(1).count_ones() <= 1,
            "RSTMGR CPU_INFO_CTRL reg INDEX field mask is not contiguous");
};

```